test/integration/testdata/golden.v2.json

```diff
@@ -403,6 +479,70 @@
      }
     }
    },
+   "maptype.AtomicMap": {
+    "type": "object",
+    "required": [
+     "KeyValue"
+    ],
+    "properties": {
+     "KeyValue": {
+      "type": "object",
+      "additionalProperties": {
+       "type": "string"
+      },
+      "x-kubernetes-map-type": "atomic"
+     }
+    }
+   },
+   "maptype.GranularMap": {
+    "type": "object",
+    "required": [
+     "KeyValue"
+    ],
+    "properties": {
+     "KeyValue": {
+      "type": "object",
+      "additionalProperties": {
+       "type": "string"
+      },
+      "x-kubernetes-map-type": "granular"
```

Comment:
The doc above says that we don't generate an openapi extension on the granular case?

Reply:
Which doc are you referring to? 🤔 (happy to change it, just to make sure I understand)